pkg/platform/aws/aws.go

```diff
@@ -74,6 +74,7 @@ type config struct {
 	HostCIDR                 string            `hcl:"host_cidr,optional"`
 	PodCIDR                  string            `hcl:"pod_cidr,optional"`
 	ServiceCIDR              string            `hcl:"service_cidr,optional"`
+	SetCSIDriverIAMRole      bool              `hcl:"set_csi_driver_iam_role,optional"`
```

Comment:
This feels too "technical" to me. We should think about flags from the eyes of the user. I suggest naming the knob `enable_csi`.